assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.js

```diff
@@ -24,6 +24,10 @@
             checkAdminUpgrade: checkAdminUpgrade
         };
 
+        // On page load we need to check to see if the upgrade is already running to set
+        // the button status and the update check running
+        checkAdminUpgrade();
```

Comment:
Could you please use an activate() function here?
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#controller-activation-promises
Also, to keep consistency with the Nodes Repo checks controller:
https://github.com/crowbar/crowbar-ui/pull/58/files#diff-2230c16354311fde135c40b1f3902a44R72